java/arcs/core/storage/StoreManager.kt

```diff
@@ -11,25 +11,23 @@
 package arcs.core.storage
 
 import arcs.core.crdt.CrdtData
-import arcs.core.crdt.CrdtOperationAtTime
-import arcs.core.util.guardedBy
-import kotlinx.coroutines.sync.Mutex
-import kotlinx.coroutines.sync.withLock
+import arcs.core.crdt.CrdtOperation
+import java.util.concurrent.ConcurrentHashMap
```

Comment:
ConcurrentHashMap is part of the Java standard library, so it's not going to compile to JS/Wasm...

Reply:
ah, right :(